libs/providers/flagd-web/src/lib/flagd-web-provider.spec.ts

```diff
@@ -207,6 +208,23 @@ describe(FlagdWebProvider.name, () => {
           type: EVENT_CONFIGURATION_CHANGE,
         });
       });
+
+      it('should trigger call to resolveAll with current context', (done) => {
+
+        client.addHandler(ProviderEvents.ConfigurationChanged, () => {
+          try {
+            expect(mockPromiseClient.resolveAll).toHaveBeenLastCalledWith({context: Struct.fromJson(context as JsonValue)});
+            done();
+          } catch(err) {
+            done(err);
+          }
+        });
+        OpenFeature.setContext(context).then(() => {
+          mockCallbackClient.mockMessage({
+            type: EVENT_CONFIGURATION_CHANGE,
+          });
+        });
+      });
```

Comment:
@beeme1mr this test fails if my fix is reverted (the context is not up-to-date).